src/main/src/main/java/com/shevtsov/dao/impl/ClientDBDao.java

```diff
@@ -0,0 +1,155 @@
+package com.shevtsov.dao.impl;
+
+import com.shevtsov.dao.ClientDao;
+import com.shevtsov.domain.Client;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClientDBDao implements ClientDao {
+    private static final ClientDao INSTANCE = new ClientDBDao();
+
+    private ClientDBDao() {
+        try (Connection connection = DriverManager.getConnection(DBCostants.DB_URL, DBCostants.LOGIN,
+                DBCostants.PASSWORD); Statement statement = connection.createStatement()) {
+            try(ResultSet resultSet = statement.executeQuery("SHOW TABLES;");){
+                while (resultSet.next()){
+                    if (resultSet.getString(1).equals("CLIENT")){
+                        return;
+                    }
+                }
+            }
+            statement.executeQuery("CREATE TABLE IF NOT EXISTS CLIENT (ID BIGINT PRIMARY KEY AUTO_INCREMENT," +
```

Comment:
If you use 'if not exist' in query, you can just execute this query without checking

Reply:
ok